modules/services/deeplink/src/main/kotlin/au/com/shiftyjelly/pocketcasts/deeplink/DeepLink.kt

```diff
@@ -244,6 +244,10 @@ data class SignInDeepLink(
     }
 }
 
+data class PocketCastsWebsiteReferralDeepLink(
```

Comment:
Names seems a bit wordy. Maybe just `ReferralsDeepLink`?

Reply:
Good point, I have renamed this.